src/main/java/org/ibp/api/rest/feedback/FeedbackResponse.java

```diff
@@ -0,0 +1,41 @@
+package org.ibp.api.rest.feedback;
+
+import org.generationcp.middleware.pojos.workbench.feedback.FeedbackFeature;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+@AutoProperty
+public class FeedbackResponse {
+
+	private FeedbackFeature feedbackFeature;
```

Comment:
Does it make more sense to edit FeedbackFeature and include collectorId there? 
Seems like we do not need this new class
@cuenyad